app/services/candidate_interface/section_policy.rb

```diff
@@ -26,7 +26,15 @@ def self.editable_sections
     end
 
     def can_edit?
-      any_offer_accepted? || all_applications_unsubmitted? || editable_section?
+      five_days_after_first_submission? ||
+        any_offer_accepted? ||
+        all_applications_unsubmitted? ||
+        editable_section?
+    end
+
+    def five_days_after_first_submission?
+      @current_application.submitted_at.present? &&
+        5.business_days.after(@current_application.submitted_at).end_of_day >= Time.zone.now
```

Comment:
We should extract this `5` to a constant. And change the method name from referencing the specific value `five`

Maybe `Rails.application.config.x.editable_window = 5`
https://guides.rubyonrails.org/configuring.html#custom-configuration